scripts/main/battle_taunt.lua

```diff
@@ -0,0 +1,60 @@
+function Mod:initBattleTaunt()
+    self.taunt_cooldown = 0
+    self.state_blacklist = {
+        "DEFENDINGBEGIN",
+        "DEFENDING",
+        "DEFENDINGEND",
+        "ENEMYDIALOGUE",
+        "ATTACKING",
+        "ACTIONS",
+        "ACTIONSDONE",
+        "INTRO",
+        "TRANSITION",
+        "TRANSITIONOUT"
+    }
+end
+
+function Mod:updateBattleTaunt()
+    local toque_equipped = false
+    for _,party in ipairs(Game.party) do
+        if party:checkArmor("pizza_toque") then toque_equipped = true end
+    end
+    if
+        (toque_equipped or Game.save_name:upper() == "PEPPINO" or self.let_me_taunt)
+        and Input.pressed("v", false)
+        and self.taunt_cooldown == 0
+        and (Game.state == "BATTLE" and Game.world.state == "GAMEPLAY" and not Game.world:hasCutscene())
```

Comment:
TBH checking Game.world.state and Game.world:hasCutscene is pretty pointless here

Reply:
forgot to remove those, whoops